osu.Game/Rulesets/RulesetInfo.cs

```diff
@@ -17,6 +18,9 @@ public class RulesetInfo : IEquatable<RulesetInfo>
 
         public string InstantiationInfo { get; set; }
 
+        [NotMapped]
```

Comment:
Not sure if this is better in the `OsuDbContext` builder?

Reply:
Given my other comment, this shouldn’t exist in RulesetInfo.

Reply:
like `Beatmap.Ruleset.CreateInstance().PlayingVerb`?

Reply:
`Ruleset.CreateInstance().PlayingVerb`, as been told, the game-wide ruleset bindable (which has been probably used here) is going to change into the instance type (`Ruleset`) by #7171.